SERVICES.md

```diff
@@ -0,0 +1,457 @@
+# Azure services and package import paths
+
+All package paths begin with **github.com/Azure/azure-sdk-for-go/services/**.
+
+<table>
+  <tr><th>
+  Service 
+    </th><th>
+  Packages
+  </th></tr>
+
+  <tr><td>
+  Advisor
+    </td><td>
+  "advisor/mgmt/2016-07-12-preview/advisor"<br />
+  "advisor/mgmt/2017-03-31/advisor"<br />
+  "advisor/mgmt/2017-04-19/advisor"<br />
+  </td></tr>
+
+  <tr><td>
+  Analysis Services
+    </td><td>
+  "analysisservices/mgmt/2016-05-16/analysisservices"<br />
+  "analysisservices/mgmt/2017-07-14/analysisservices"<br />
+  "analysisservices/mgmt/2017-08-01-beta/analysisservices"<br />
+  </td></tr>
+
+  <tr><td>
+  API Management
+    </td><td>
+  "apimanagement/mgmt/2016-10-10/apimanagement"
+  </td></tr>
+
+  <tr><td>
+  App Insights
+    </td><td>
+  "appinsights/mgmt/2015-05-01/insights"
+  </td></tr>
+
+  <tr><td>
+  Authorization
+    </td><td>
+  "authorization/mgmt/2015-07-01/authorization"
+  </td></tr>
+
+  <tr><td>
+  Automation
+    </td><td>
+  "automation/mgmt/2015-10-31/automation"
+  </td></tr>
+
+  <tr><td>
+  Batch
+    </td><td>
+  "batch/mgmt/2015-12-01/batch"<br />
+  "batch/mgmt/2017-01-01/batch"<br />
+  "batch/mgmt/2017-05-01/batch"<br />
+  "batch/mgmt/2017-09-01/batch"<br />
+  </td></tr>
+
+  <tr><td>
+  Billing
+    </td><td>
+  "billing/mgmt/2017-02-27-preview/billing"<br />
```

Comment:
Would it be too much effort to make these links? :)

Reply:
a reasonable request but I'd like to defer since it's error-prone. for now the main purpose of this page is awareness of what's required to select a version so we can get feedback. if this sticks do you think we could automate creation of columns for this table via [profileBuilder](https://github.com/Azure/azure-sdk-for-go/tree/master/tools/profileBuilder)?